### ISSUE TITLE
Use NAT rules to override visibility

### ISSUE THREAD

**Issue by simmhan** [dream-lab/VIoLET#6]

**Comment by shreyas-badiger** [comment]
Should this be taken as an input from the user? At what level must the visibility restriction be enforced? Between devices in public network?

**Comment by simmhan** [comment]
Yes, input in infra JSON. Visibility *restriction* should be specified on a per-device pair basis in public networks. Should try and make it asymmetric -- D1 can talk to D2, but D2 not to D1.

This issue is also related to the default being set to all-to-all access for all public devices.

Later, we can think about extending feature to private networks as well.

**Comment by simmhan** [comment]
IP table rules allow this
